### PR TITLE
adding cdt dataset

### PR DESCRIPTION
**Pull request by abecadel**
- **Name:** *Cyberbullying Detection Task*
- **Description:** *The Cyberbullying Detection task was part of 2019 edition of PolEval competition. The goal is to predict if a given Twitter message contains a cyberbullying (harmful) content.*
- **Data:** *https://github.com/ptaszynski/cyberbullying-Polish*
- **Motivation:** *The KLEJ benchmark (Kompleksowa Lista Ewaluacji Językowych) is a set of nine evaluation tasks for the Polish language understanding.*

### CODE REVIEW

File: datasets/cdt/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+YAML tags:
+- copy-paste the tags obtained with the tagging app: http://34.68.228.168:8501/
```
Comment (lhoestq) [inline datasets/cdt/README.md:3]: can you fill the tags using the tagger please ?

File: datasets/cdt/cdt.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+"""
+
+_HOMEPAGE = "https://github.com/ptaszynski/cyberbullying-Polish"
+
+_LICENSE = "BSD 3-Clause"
+
+_URLs = "https://klejbenchmark.com/static/data/klej_cbd.zip"
+
+
+class CyberbullyingDetectionTask(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/cdt/cdt.py:47]:
```suggestion
class Cdt(datasets.GeneratorBasedBuilder):
```
this needs to be the camel case version of the dataset script name `cdt`

File: datasets/cdt/cdt.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    VERSION = datasets.Version("1.1.0")
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "sentence": datasets.Value("string"),
+                    "target": datasets.Value("float"),
```
Comment (lhoestq) [inline datasets/cdt/cdt.py:58]: can you use the `ClassLabel` feature type for `target` ?
for example `ClassLabel(["neg", "pos"])`

**Review by lhoestq — COMMENTED** [review]
Thanks for adding this one !

I left a few comments

after the change you'll need to regenerate the dataset_infos.json file as well

File: datasets/cdt/README.md (hunk 1 above)
Comment (abecadel) [reply datasets/cdt/README.md]: done

File: datasets/cdt/cdt.py (hunk 3 above)
Comment (abecadel) [reply datasets/cdt/cdt.py]: done

**Comment by abecadel** [comment]
> Thanks for adding this one !
> 
> I left a few comments
> 
> after the change you'll need to regenerate the dataset_infos.json file as well

dataset_infos.json regenerated

**Comment by lhoestq** [comment]
looks like this PR includes changes to many files other that the ones for CDT
could you create another branch and another PR please ?

**Pull request closed without merging**